### DISCUSSION TITLE
How to install postgres_lsp on the local system using any package manager?

### DISCUSSION THREAD

**Discussion by sujayxaradhya** [supabase-community/postgres-language-server#41] [General]
I was wondering, How to install postgres_lsp using npm or homebrew? I'm a MacOS user and Helix editor(inspired by neovim and kakoune) user. I literally need this LSP to continue working on postgresql. Any help would really be appreciated.

**Comment by DanielFGray** [comment]
`postgres_lsp` currently only handles very simple queries, it will not be very useful for general-purpose development

if you're desperate enough and have the budget, Jetbrains' DataGrip has the desired functionality you're looking for in the meantime

**Reply by psteinroe** [reply]
thanks for the answer @DanielFGray! Right now, the only way to use it is to build it from source and point to the executable. As already pointed out, we do not recommend using it yet though.
